feat(cli): add `--formatter ` and `--linter` option to `biome rage` (#1843)
biomejs/biome: CHANGELOG.md

- Now the file name printed in the diagnostics is clickable. If you run the CLI from your editor, you can <kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd> + Click on the file name, and the editor will open said file. If row and columns are specified e.g. `file.js:32:7`, the editor will set the cursor right in that position. Contributed by @ematipico

> - Add option `--linter` to `biome rage`. The option is required to view Biome linter configuration.  Contributed by @seitarof

Reviewer: ```suggestion
- Add an option `--linter` to `biome rage`. The option needs to check Biome linter configuration.  Contributed by @seitarof
```

Author: fixed 306d88b

Author: Fixed cd29d1b
